content/docs/0.8.0/users-guide/default-disk-and-node-config.md

```diff
@@ -1,41 +1,78 @@
-# Default disks and node configuration
+---
+title: Configuring Defaults for Nodes and Disks
+weight: 1
+---
 
-## Overview
+_Available as of v0.8.1_
 
-This feature allows the user to customize the default disks and node configurations in Longhorn for newly added nodes **using Kubernetes label and annotation**, instead of using Longhorn API or UI.
+This feature allows the user to customize the default disks and node configurations in Longhorn for newly added nodes using Kubernetes labels and annotations instead of the Longhorn API or UI.
 
-This feature is available since v0.8.1.
+Customizing the default configuations for disks and nodes is useful for scaling the cluster because it eliminates the need to configure Longhorn manually for each new node if the node contains more than one disk, or if the disk configuration is different for new nodes.
 
-## Usage
+> **Note:** Longhorn will not keep the node labels or annotations in sync with the current Longhorn node disks or tags.
 
-### Add node tags for new nodes
+### Adding Node Tags to New Nodes
 
-1. Scale up the Kubernetes cluster. Then the newly added nodes contain no node tag.
-2. Add the annotations to the new Kubernetes nodes. 
-    - The annotation format is `node.longhorn.io/default-node-tags: <node tag list with JSON string format>`. For example:
+When a node does not have a tag, Longhorn uses a node annotation to set the node tags.
```

Comment:
`user can use a node annotation to set the node tags, as an alternative to using the Longhorn UI/API`. Since the user can also use Longhorn UI to set it.

Reply:
changed to:

> When a node does not have a tag, you can use a node annotation to set the node tags, as an alternative to using the Longhorn UI or API.